### PR TITLE
Format EnsurancePolicy to change *int to int

### PR DESCRIPTION
**Pull request by yan234280533**

### CODE REVIEW

File: artifacts/deploy/ensurance.crane.io_avoidanceactions.yaml [hunk 1]
```diff
@@ -38,7 +38,7 @@ spec:
               coolDownSeconds:
                 default: 300
                 description: CoolDownSeconds is the seconds for cool down when do
-                  avoidance default is 300s
+                  avoidance Default is 300s. Minimum value is 1.
```
Comment (mfanjie) [inline artifacts/deploy/ensurance.crane.io_avoidanceactions.yaml:41]: please take care of the description format, seems several lines are merged.
and please note the format
Default is 300, Minimum value is 1 should be
`Default value is 300 and Minimum value is 1`
or `Default is 300s and Minimum is 1s`

Comment (mfanjie) [reply artifacts/deploy/ensurance.crane.io_avoidanceactions.yaml:41]: and there are other description `Default to 1` and `Default is 300s`
suggest to make them identical.

Comment (yan234280533) [reply artifacts/deploy/ensurance.crane.io_avoidanceactions.yaml]: done~

**Pull request merged**